.konflux/operator/konflux.Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG OPERATOR_VERSION
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.23@sha256:4805e1cb2d1bd9d3c5de5d6986056bbda94ca7b01642f721d83d26579d333c60 as builder
+
+WORKDIR /go/src/github.com/openshift-kni/numaresources-operator
+COPY . .
+
+ENV GOEXPERIMENT=strictfipsruntime
+ENV CGO_ENABLED=1
+ENV GOTAGS="strictfipsruntime"
+
+# Build
+RUN \
+	NRO_BUILD_VERSION=${OPERATOR_VERSION} \
+	make binary-all
+
+FROM registry.redhat.io/rhel9-4-els/rhel-minimal:9.4@sha256:65e57c845402711c5515af0989a2c3c69bf4066396008efd8002be0790fee6c3
+
+COPY --from=builder /go/src/github.com/openshift-kni/numaresources-operator/bin/manager /bin/numaresources-operator
+# bundle the operand, and use a backward compatible name for RTE
+COPY --from=builder /go/src/github.com/openshift-kni/numaresources-operator/bin/exporter /bin/resource-topology-exporter
+COPY --from=builder /go/src/github.com/openshift-kni/numaresources-operator/bin/buildinfo.json /usr/local/share
+
+RUN mkdir /etc/resource-topology-exporter/ && \
+    touch /etc/resource-topology-exporter/config.yaml
+RUN microdnf install -y hwdata && \
+    microdnf clean -y all
+USER 65532:65532
+ENTRYPOINT ["/bin/numaresources-operator"]
+LABEL com.redhat.component="numaresources-operator-container" \
+      name="openshift4/numaresources-operator-rhel9" \
+      version="v${OPERATOR_VERSION}" \
+      summary="numaresources-operator" \
+      io.openshift.expose-services="" \
+      io.openshift.tags="operator" \
+      io.k8s.display-name="numaresources-operator" \
+      io.k8s.description="Numa Resources Operator" \
+      maintainer="openshift-operators@redhat.com" \
+      description="numaresources-operator" \
+      io.openshift.maintainer.component="NUMA Resources Operator" \
+      io.openshift.maintainer.product="OpenShift Container Platform" \
+      distribution-scope="public" \
+      release="${OPENSHIFT_VERSION}" \
```

Comment:
IIUC this is defined in https://github.com/openshift-kni/numaresources-operator/pull/1272/files#diff-7b7d730ccb067e0b4b4b41a95f9e988f92f4dee60187ef0e45f30a8750950b71R2, don't we want to pass it to the dockerfile as ARG same as the OPERATOR_VERSION?

Reply:
good point, since I am not building the must gather image with this PR I missed it, I will add it here.